Shared/Objects/VideoPlayerJumpLength.swift

```diff
@@ -0,0 +1,59 @@
+//
+ /* 
+  * SwiftFin is subject to the terms of the Mozilla Public
+  * License, v2.0. If a copy of the MPL was not distributed with this
+  * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+  *
+  * Copyright 2021 Aiden Vigue & Jellyfin Contributors
+  */
+
+import UIKit
+
+enum VideoPlayerJumpLength: Int32, CaseIterable {
```

Comment:
In [Swift Docs](https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html#ID319])

>Int
>In most cases, you don’t need to pick a specific size of integer >to use in your code. Swift provides an additional integer type, >Int, which has the same size as the current platform’s native word >size:
>
>On a 32-bit platform, Int is the same size as Int32.
>On a 64-bit platform, Int is the same size as Int64.
>Unless you need to work with a specific size of integer, always >use Int for integer values in your code. This aids code >consistency and interoperability. Even on 32-bit platforms, Int can store any value between -2,147,483,648 and 2,147,483,647, and is large enough for many integer ranges.

If you are using `Int32` because the value is fixed, `UInt8` is more suitable lol
However I recommend using `Int`.

Reply:
And I recommend creating a property such as `length` rather than using rawValue.

```swift
var length: Int {
switch self {
	case thirty:
	return 30
	...
}
```

Reply:
It needs to be Int32 because that's what the media player needs, I wouldn't be using it otherwise.
I'm fine to change it to `length` variable.

Reply:
> It needs to be Int32 because that's what the media player needs, I wouldn't be using it otherwise.

Oh, I understand.